packages/tools/client-debug-view/src/components/AudienceView.tsx

```diff
@@ -162,20 +162,45 @@ function HistoryView(props: HistoryViewProps): React.ReactElement {
 	const reversedHistoryLog = [...history].reverse();
 
 	const historyViews: React.ReactElement[] = [];
+
 	for (const changeEntry of reversedHistoryLog) {
 		const changeTimeStamp = new Date(changeEntry.timestamp);
 		const wasChangeToday = nowTimeStamp.getDate() === changeTimeStamp.getDate();
 
+        const accordianBackgroundColor: IStackItemStyles = {
+            root: {
+                background: changeEntry.changeKind === "added" ? "#90ee90" : "#FF7377",
+                borderStyle: "solid",
+                borderWidth: 1,
+                borderColor: DefaultPalette.neutralTertiary,
+                padding: 3
+            }
+        }
+
+        const iconStyle: IStackItemStyles = {
+            root: {
+                padding: 10
+            }
+        }
+
 		historyViews.push(
-			<li key={`${changeEntry.clientId}-${changeEntry.changeKind}`}>
-				<b>Client ID: </b>
-				{changeEntry.clientId}
-				<br />
-				<b>Time: </b>{" "}
-				{wasChangeToday ? changeTimeStamp.toTimeString() : changeTimeStamp.toDateString()}
-				<br />
-				<b>Type: </b> {changeEntry.changeKind}
-			</li>,
+            <div>
```

Comment:
Nit: do we need this outer `div`? I think `Stack` is rendered as a `div` anyways.

Reply:
I inserted the outer `div` to differentiate between `audience` and `historyViewer`. 

 Since we will be leaving `overflowY: "auto"` for `<Stack>` tag, I tried to add `scroll` style to `{historyViewer}` by creating an outer `div` tag.

Reply:
But these are the individual history elements, not the outer audience container element. I _think_ the outer div here is redundant, since we aren't applying any styling to it. Since Stack is rendered as a div, I think the resulting tree structure for each of the entries here has 2 levels of outer div, when they only need 1 (unless I'm misunderstanding something).